[DEVX-161] Improve finance api docs (#118)
iZettle/api-documentation: finance-api/user-guides/fetch-account-balance.md

Fetch account balance
===
Use the Finance API to fetch the account balance of the merchant's preliminary or liquid account.

* [Prerequisites](#prerequisites)
* [Fetch the account balance](#fetch-the-account-balance)
* [Related task](#related-task)
* [Related API reference](#related-api-reference)

## Prerequisites
* Make sure that authorization is set up with the required OAuth scope using [Authorization OAuth2 API](../../authorization.adoc).

## Fetch the account balance 
Fetch the balance in one or both of the merchant's preliminary and liquid accounts as follows:
   * To check the balance of transactions that are still being checked by Zettle whether they should be paid out, fetch the preliminary account balance:
     ```
     GET /organizations/self/accounts/preliminary/balance
     ```
   * To check the balance of transactions that are ready to be paid out to the merchant, fetch the liquid account balance:
     ```
     GET /organizations/self/accounts/liquid/balance

>      ```

Reviewer: This is perfect now!

Fetch account balance
===
Use the Finance API to fetch the account balance of the merchant's preliminary or liquid account.

* [Prerequisites](#prerequisites)
* [Fetch the account balance](#fetch-the-account-balance)
* [Related task](#related-task)
* [Related API reference](#related-api-reference)

## Prerequisites
* Make sure that authorization is set up with the required OAuth scope using [Authorization OAuth2 API](../../authorization.adoc).  
<!-- to be continued if any -->

## Fetch the account balance 
Fetch the balance in one or both of the merchant's preliminary and liquid accounts as follows:

>    * To check the balance of transactions that are still being cleared by the acquiring bank, fetch the preliminary account balance:

Reviewer: This is not correct. Transactions that are in the preliminary is because they weren't liquefied yet as we wrote in the documentation. The transaction goes from preliminary to liquid when:

```
Zettle checks whether the payment is ready to be paid out to the merchant.
```

So the correct here would be something like
```
To check the balance of transactions that are still being checked by Zettle whether they should be paid out or not, fetch the preliminary account balance:
```

Then next step is correct, which says that liquid balance is what is ready to be paid out.

Fetch account balance
===

> Using the Finance API, you can fetch the balance that is available in the merchant's Zettle preliminary or liquid account.

Reviewer 2: I think we can drop "that is available" to simplify.
Balance is always available 

```suggestion
Use the Finance API to fetch the account balance of the the merchant's preliminary or liquid account.
```

Fetch account balance
===
Using the Finance API, you can fetch the balance that is available in the merchant's Zettle preliminary or liquid account.

* [Prerequisites](#prerequisites)
* [Step 1: Fetch the account balance](#step-1-fetch-the-account-balance)
* [Related task](#related-task)
* [Related API reference](#related-api-reference)

## Prerequisites
* Make sure that authorization is set up using [Authorization OAuth2 API](../../authorization.adoc). 
<!-- to be continued if any -->

> ## Step 1: Fetch the account balance 

Reviewer 2: Do we need to have step nr when there is only one step?

Fetch the account balance in the merchant's Zettle preliminary or liquid account.  

> 1. Optional: Fetch your organisation UUID. 

Reviewer 2: Why complicate it with `users/me` when we have orgUuid in the token already and can use `self`?

   > **Tip:** You don't need to fetch the organisation UUID, as you can specify the organisation UUID as `self` in requests that require it.

    ```
    GET users/me
    ```
   Example:
       
   The following example response returns organisation UUID `ab305d54-75b4-431b-adb2-eb6b9e546013`.

    ```
    {
        "uuid": "de305d54-75b4-431b-adb2-eb6b9e546014",
        "organizationUuid": "ab305d54-75b4-431b-adb2-eb6b9e546013"
    }
    ```

> 2. Fetch the balance in one or both of the merchant's preliminary and liquid accounts as follows:

Reviewer 2: We need to check whether we should use _merchant_ or _seller_

   > **Tip:** You specify the organisation UUID as `self`.

>    * To check the balance of transactions that are still being confirmed by the third-party acquirer with the buyer's bank, fetch the preliminary account balance:

Reviewer 2: ```suggestion
   * To check the balance of transactions that are still in the process of being cleared by the acquirer, fetch the preliminary account balance:
```

     ```
     GET /organizations/{organizationUuid}/accounts/preliminary/balance
     ```

>    * To check the balance of transactions that are ready to be paid out to the merchant's bank, fetch the liquid account balance:

Reviewer 2: Since money can be paid out to merchant's PayPal wallet
```suggestion
   * To check the balance of transactions that are ready to be paid out to the merchant, fetch the liquid account balance:
```

     ```
     GET /organizations/{organizationUuid}/accounts/liquid/balance
     ```

   Example:
   
   The following example request fetches account balance in the merchant's Zettle preliminary account.
   
   ```
   GET /organizations/self/accounts/preliminary/balance
   ```
       

>    The following example response shows that £100 is still being confirmed by the third-party acquirer with the buyer's bank.

Reviewer 2: We record amounts in minor currency units.
```suggestion
   The following example response shows that £1 has not yet got cleared by the acquirer.
```

Fetch account balance
===
Using the Finance API, you can fetch the balance that is available in the merchant's Zettle preliminary or liquid account.

* [Prerequisites](#prerequisites)
* [Step 1: Fetch the account balance](#step-1-fetch-the-account-balance)
* [Related task](#related-task)
* [Related API reference](#related-api-reference)

> ## Prerequisites

Reviewer 3: What about adding the necessary scope to all these prerequisites?

    }
    ```

2. Fetch the balance in one or both of the merchant's preliminary and liquid accounts as follows:
   > **Tip:** You specify the organisation UUID as `self`.
   * To check the balance of transactions that are still being confirmed by the third-party acquirer with the buyer's bank, fetch the preliminary account balance:
     ```
     GET /organizations/{organizationUuid}/accounts/preliminary/balance
     ```
   * To check the balance of transactions that are ready to be paid out to the merchant's bank, fetch the liquid account balance:
     ```
     GET /organizations/{organizationUuid}/accounts/liquid/balance
     ```

   Example:
   
   The following example request fetches account balance in the merchant's Zettle preliminary account.
   
   ```
   GET /organizations/self/accounts/preliminary/balance
   ```
       
   The following example response shows that £100 is still being confirmed by the third-party acquirer with the buyer's bank.

    ```json
    {
        "data": {
            "totalBalance": 100,
            "currencyId": "GBP"
        }
    }    

 
## Related task
* [Fetch account transactions](fetch-account-transactions)

> * [Fetch payout information](fetch-payout-info)

Reviewer 3: Links not working (missing `.md`?)

Author: Thanks! Fixed all broken links now.
